### DISCUSSION TITLE
Auto-determine profile based on account

### DISCUSSION THREAD

**Discussion by asmith-personal** [runelite/runelite#17178] [Ideas]
I use Jagex Launcher with a Jagex Account. As someone who frequently has multiple accounts running, and has a separate RuneLite Profile per account,  it would be great if RuneLite could determine on-launch the appropriate profile to use. It's a bit frustrating to have to load wrong_profile's plugins as RL launches, then realize I'm on the wrong profile and effectively have to reload everything when I switch into correct_profile

One possible solution could be a configurable toggle in the profiles "if Profile name matches RSN, use that profile."

So when Jagex Launcher launches the account Zezima's RuneLite, RuneLite will search for and auto-switch to the Zezima RuneLite profile (if it exists). Alternatively, someone smarter than me could think of a configuration menu where you can correlate "when I launch this account, use this profile."

**Comment by Alexsuperfly** [comment]
https://runelite.net/blog/show/2023-11-05-1.10.15-Release

**Reply by Nightfirecat** [reply]
We also plan to extend this feature to automatically load the correct profile when using the Jagex launcher, as we should have the necessary information at client launch (not just at login time) to choose the appropriate profile.

**Comment by asmith-personal** [comment]
OOF ty @Alexsuperfly

**Comment by fixshades** [comment]
anymore info on this?
